### PR TITLE
COMMERCE-2282

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test

File: commerce-product-service/src/main/java/com/liferay/commerce/product/internal/util/CPVersionContributorRegistryImpl.java [hunk 1]
```diff
@@ -67,9 +68,15 @@ public CPVersionContributor getCPVersionContributor(String key) {
 			cpVersionContributorServiceWrappers = ListUtil.fromCollection(
```
Comment (ethanbustad) [inline commerce-product-service/src/main/java/com/liferay/commerce/product/internal/util/CPVersionContributorRegistryImpl.java]: We probably don't need to turn this into a list before iterating over it now, yeah? Saves us an object and a bit of unnecessary processing.

File: commerce-product-service/src/main/java/com/liferay/commerce/product/internal/util/CPVersionContributorRegistryImpl.java [hunk 2]
```diff
@@ -67,9 +68,15 @@ public CPVersionContributor getCPVersionContributor(String key) {
...
 				_serviceTrackerMap.values());
 
-		List<CPVersionContributor> cpVersionContributors = ListUtil.toList(
-			cpVersionContributorServiceWrappers,
-			ServiceWrapper<CPVersionContributor>::getService);
+		List<CPVersionContributor> cpVersionContributors = new ArrayList<>();
```
Comment (ethanbustad) [inline commerce-product-service/src/main/java/com/liferay/commerce/product/internal/util/CPVersionContributorRegistryImpl.java]: We could also initialize this list with the size of the `cpVersionContributorServiceWrappers`, for a little more efficiency.

**Review by ethanbustad — COMMENTED** [review]
A couple very minor performance-y things to think about. @marco-leo definitely feel free to merge first if we need this asap, as these are not essential changes in my mind.

**Comment by marco-leo** [comment]
@Commando18 please make the changes Ethan is asking and I will merge the PR

**Comment by alessiorendina** [comment]
ci:test

**Comment by alessiorendina** [comment]
@marco-leo @ethanbustad Updated, thanks!

**Pull request merged**